website/docs/providers/proxy/forward_auth.mdx

```diff
@@ -2,19 +2,14 @@
 title: Forward auth
 ---
 
-import Tabs from "@theme/Tabs";
-import TabItem from "@theme/TabItem";
-
-Using forward auth uses your existing reverse proxy to do the proxying, and only uses the
-authentik outpost to check authentication and authorization.
+Using forward auth uses your existing reverse proxy to do the proxying, and only uses the authentik outpost to check authentication and authorization.
 
 To use forward auth instead of proxying, you have to change a couple of settings.
 In the Proxy Provider, make sure to use one of the Forward auth modes.
```

Comment:
Are there docs we can link to, that explain procedure for configuring the proxy provider to use forward auth mode?

Reply:
that's what these docs are, but in the web UI, there's just a toggle switch between the modes when editing a proxy provider